test/unit_tests/utils/calculation_test.js

```diff
@@ -0,0 +1,26 @@
+/* global describe it */
+
+const chai = require( 'chai' );
+const expect = chai.expect;
+const calculation = require( '../../../src/js/utils/calculation' );
+
+describe( 'calculation', () => {
+
+  describe( 'getFirstNumber', () => {
+
+    it( 'should return the first number in an array of values', () => {
+      const valToCheck = 2;
+      const mockArray = [
+        [ 'one', 'one' ],
```

Comment:
This test does not work as expected. If I add another number value to this first array (for example, `[ 'one', 'one', 4 ]`), I would expect the test to fail, but it still passes.

Reply:
if this is because the function just works this way for now, then a note / todo here would be friendly

Reply:
The method itself has several TODOs, is that enough? https://github.com/cfpb/cfpb-chart-builder/pull/121/files#diff-f543c19dcfe072e9cf7ddce186eff78cR3

Reply:
Since the underlying behavior of the method is the issue, not the test.

Reply:
Yeah those are good for now, maybe just make the part about structure more specific for what this function does and doesn't do for the time being